e2e/framework/utils/k8s_helper.go

```diff
@@ -304,3 +286,45 @@ func (k8sh *K8sHelper) GetPodHostId(podNamePattern string, namespace string) (st
 		return serr, fmt.Errorf("Error Getting Monitor IP")
 	}
 }
+
+func (k8sh *K8sHelper) IsStorageClassPresent(name string) (bool, error) {
+	cmdArgs := []string{"get", "storageclass", "-o", "jsonpath='{.items[*].metadata.name}'"}
+	sout, serr, _ := ExecuteCmd("kubectl", cmdArgs)
+	if strings.Contains(sout, name) {
+		return true, nil
+	}
+	return false, fmt.Errorf("Storageclass %s not found, err ->%s", name, serr)
+
+}
+
+func (k8sh *K8sHelper) GetPVCStatus(name string) (string, error) {
+	cmdArgs := []string{"get", "pvc", "-o", "jsonpath='{.items[*].metadata.name}'"}
+	sout, serr, _ := ExecuteCmd("kubectl", cmdArgs)
+	if strings.Contains(sout, name) {
+		cmdArgs := []string{"get", "pvc", name, "-o", "jsonpath='{.status.phase}'"}
+		res, _, _ := ExecuteCmd("kubectl", cmdArgs)
+		return res, nil
+	}
+	return "PVC NOT FOUND", fmt.Errorf("PVC %s not found,err->%s", name, serr)
```

Comment:
I made a note of your earlier comment, We are going to clean up these error message in one go.